test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/AwsSqsTests.cs

```diff
@@ -0,0 +1,174 @@
+// <copyright file="AwsSqsTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Core.Tools;
+using Datadog.Trace.ClrProfiler.IntegrationTests.Helpers;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AWS
+{
+    public class AwsSqsTests : TestHelper
+    {
+        private static List<MockTracerAgent.Span> _expectedSpans = new()
+        {
+#if NETFRAMEWORK
+            // CreateSqsQueue
+            AwsSqsSpan.GetDefault("CreateQueue")
+                    .WithTag("aws.queue.name", "MySyncSQSQueue"),
+            AwsSqsSpan.GetDefault("CreateQueue")
+                    .WithTag("aws.queue.name", "MySyncSQSQueue2"),
+
+            // SendMessagesWithInjectedHeaders
+            AwsSqsSpan.GetDefault("SendMessage"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+
+            // SendMessagesWithoutInjectedHeaders
+            AwsSqsSpan.GetDefault("SendMessage"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+
+            // SendMessage
+            AwsSqsSpan.GetDefault("SendMessage"),
+            AwsSqsSpan.GetDefault("SendMessage"),
+
+            // ReceiveMessageAndDeleteMessage
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+
+            // SendMessageBatch
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+
+            // ReceiveMessagesAndDeleteMessageBatch
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+
+            // DeleteQueue
+            AwsSqsSpan.GetDefault("DeleteQueue"),
+            AwsSqsSpan.GetDefault("DeleteQueue"),
+
+#endif
+            // CreateSqsQueueAsync
```

Comment:
May be worth adding a note reflecting [the explanation in your comment](https://github.com/DataDog/dd-trace-dotnet/pull/1471#discussion_r638171112)? Just so it's clearer it's not a typo? 🙂 

```suggestion
            // Span resource names do not include the Async prefix
            // CreateSqsQueueAsync
```

Reply:
Fixed in d19668f4916196542b7a0f7329193a9f80499c53